src/app/integrations/edit-page/configure-action/configure-action.component.ts

```diff
@@ -36,7 +37,7 @@ export class IntegrationsConfigureConnectionComponent implements OnInit, OnDestr
 
   handleFlowEvent(event: FlowEvent) {
     switch (event.kind) {
-      case 'integration-no-connection':
+      case 'integration-no-action':
```

Comment:
It's called `integration-no-actions` in the `edit-page.component.ts` file. I'll change that now.